### PR TITLE
add expanded ios/xcode documentation

### PR DESCRIPTION
**Pull request by jacobsalmela**
closes https://github.com/haji mehoshi/ebiten/issues/1906

Signed-off-by: Jacob Salmela <me@jacobsalmela.com>

### CODE REVIEW

File: contents/documents/mobile.html [hunk 1]
```diff
@@ -104,6 +104,34 @@ <h3>iOS</h3>
 
 @end</code></pre>
 
+<h4>Integrating Into Xcode</h4>
+
+Part of the appeal of using <code>ebiten</code> is that you don't have to have a lot of experience in Xcode in order to build games for iOS.  Once you import the <code>ebitenmobile</code>-generated framework and setup the View Controller within Xcode, the rest of your coding can take place in Golang.  When you need to test a new change, run the <code>ebitenmobile</code> command again and then Build your project in Xcode.
```
Comment (hajimehoshi) [inline contents/documents/mobile.html]: Use `<p>`

File: contents/documents/mobile.html [hunk 2]
```diff
@@ -104,6 +104,34 @@ <h3>iOS</h3>
...
+
+<br></br>
+
+In it's simplest form, the Xcode project will have these files:
```
Comment (hajimehoshi) [inline contents/documents/mobile.html]: ditto

File: contents/documents/mobile.html [hunk 3]
```diff
@@ -104,6 +104,34 @@ <h3>iOS</h3>
...
+  <li><code>Assets.xcassets</code></li>
+  <li><code>Base.lproj</code></li>
+  <li><code>Info.plist</code></li>
+  <li><code>EbitenViewController.h</code></li>
+  <li><code>EbitenViewController.m</code></li>
+  <li><code>main.m</code></li>
+  <li><code>Yourgame.framework</code></li>
+</ul>
+
+When you start a new project, you are given the choice of many different types of apps to create.  If you're just beginning to work with Xcode, you may want to start with a simple one like the Single View Application or Document Viewer App.  And then modifying it to use the above files.
```
Comment (hajimehoshi) [inline contents/documents/mobile.html]: ditto

File: contents/documents/mobile.html [hunk 4]
```diff
@@ -104,6 +104,34 @@ <h3>iOS</h3>
...
+
+<h5>Setting Up The View Controller</h5>
```
Comment (hajimehoshi) [inline contents/documents/mobile.html]: Can we avoid using h5?

File: contents/documents/mobile.html [hunk 5]
```diff
@@ -104,6 +104,34 @@ <h3>iOS</h3>
...
+
+While viewing <code>main.m</code> in Xcode, click the <b>Attributes Inspector</b> tab and select <b>Is Initial View Controller</b>.  This will configure your Go code to be the first thing that is run.  Many iOS apps have a separate LaunchScreen View Controller, which you can implement if desired, but the above will allow everything else to happen within your Go code.
```
Comment (hajimehoshi) [inline contents/documents/mobile.html]: ditto

File: contents/documents/mobile.html [hunk 6]
```diff
@@ -104,6 +104,34 @@ <h3>iOS</h3>
...
+
+The example below is a great reference when you start building out your own app.
```
Comment (hajimehoshi) [inline contents/documents/mobile.html]: ditto

**Review by hajimehoshi — COMMENTED** [review]
Thanks!

Please do `go generate` as the README says.

**Comment by jacobsalmela** [comment]
Ah sorry.  I was reading the ebiten README, not this one.. I'll fix things up

**Review by hajimehoshi — COMMENTED** [review]
Use `<p>` tags. Thanks!

**Comment by jacobsalmela** [comment]
OK, fixed up.

File: README.md [hunk 7]
```diff
@@ -23,3 +23,5 @@ GOOGLE_APPLICATION_CREDENTIALS=/path/to/credentials.json go run ./cmd/uploadwasm
 ```
 go run server.go ./docs
 ```
+
+Validate your changes by opening http://127.0.0.1:8000
```
Comment (hajimehoshi) [inline README.md]: Add `.`.

File: contents/documents/mobile.html [hunk 8]
```diff
@@ -104,6 +104,31 @@ <h3>iOS</h3>
 
 @end</code></pre>
 
+<h4>Integrating Into Xcode</h4>
+
+<p>Part of the appeal of using <code>ebiten</code> is that you don't have to have a lot of experience in Xcode in order to build games for iOS.  Once you import the <code>ebitenmobile</code>-generated framework and setup the View Controller within Xcode, the rest of your coding can take place in Golang.  When you need to test a new change, run the <code>ebitenmobile</code> command again and then Build your project in Xcode.</p>
```
Comment (hajimehoshi) [inline contents/documents/mobile.html]: `<code>ebiten</code>` is not a term in a code. Use `Ebiten`.

File: contents/documents/mobile.html [hunk 9]
```diff
@@ -104,6 +104,31 @@ <h3>iOS</h3>
...
+
+<p>In it's simplest form, the Xcode project will have these files:</p>
```
Comment (hajimehoshi) [inline contents/documents/mobile.html]: `its`?

File: contents/documents/mobile.html (hunk 8 above)
Comment (hajimehoshi) [inline contents/documents/mobile.html]: Replace `Golang` with `Go`.

Comment (hajimehoshi) [inline contents/documents/mobile.html]: Replace `then Build` with `then build`.

File: README.md (hunk 7 above)
Comment (jacobsalmela) [reply README.md]: Fixed

File: contents/documents/mobile.html (hunk 8 above)
Comment (jacobsalmela) [reply contents/documents/mobile.html]: Fixed

File: contents/documents/mobile.html (hunk 9 above)
Comment (jacobsalmela) [reply contents/documents/mobile.html]: Fixed

File: contents/documents/mobile.html (hunk 8 above)
Comment (jacobsalmela) [reply contents/documents/mobile.html]: Fixed

Comment (jacobsalmela) [reply contents/documents/mobile.html]: Fixed.

**Review by hajimehoshi — APPROVED** [review]
LGTM, thanks!

**Pull request merged**